### DISCUSSION TITLE
CS75%40UCSD/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#84] [Announcements]
# CS75%40UCSD/

https://csmsapp.github.io/CS75%40UCSD/

**Comment by Leolty** [comment]


**Reply by xichenpan** [reply]
也难说，感觉纯random。SJTU CS今年申请来看和pub没啥关系，和GPA也关系不大，也不知道在看啥

**Comment by zhongxx0227** [comment]
UCSD的MSCS就是CS75嘛

**Reply by xichenpan** [reply]
yep

**Reply by wmkai** [reply]
申了76 无了

**Comment by QAQdev** [comment]
想问一下UCSD ms program官网分了三个Plan，分别都对应那几个(cs75，cs76)呢

**Comment by GeorgeZhangDS** [comment]
感觉低了

**Reply by ghost** [reply]
认可，感觉可以调整到A+的，收到录取的大部分还是较好985+高GPA的，算是A档水平？但是由于申请人数太多随机性较强，可以往上走走？和卡妹的msin一级？

**Reply by Fridge003** [reply]
同意A+，这个项目和CMU MSIN，Gatech MSCS， UIUC MCS经常拿来比较

**Reply by AnnabelleMin** [reply]
同意A+

**Reply by WuzhouDu** [reply]
问题是我A+除了录了这个，GT MSCS, UT-Austin ECE SES, Duke MSCS全拒了。UIUC MCS waitlist

**Reply by godjoem** [reply]
A非常合理，录取size比A+的几个项目大太多。就24fall来看bar比GT MSCS, Austin ECE低，不少人同时申这三个项目最后只拿到UCSD

**Reply by gelinhande** [reply]
绝对比A+高 uiuc mcs3.5都能进 msin也是 西北一般是保底的

**Reply by gelinhande** [reply]
西北跟ucs一个难度

**Reply by gelinhande** [reply]
不是ucs 是usc 打错了

**Reply by xxxbrem** [reply]
对海本uiuc mcs和nu mscs都比较容易，但陆本稍难。ucsd mscs不歧视陆本，因此相对简单，我觉得a是合理的。

**Comment by ciel307** [comment]
建议上调难度，很玄学，看到不少录取A+/S的人没录上这个，而且26fall似乎减招了

**Reply by N1rVaNa30** [reply]
大失败了
